pkgs/data/fonts/dotsies/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "dotsies";
+
+  regular = fetchurl {
+    name = "dotsies-regular";
+    url = "http://dotsies.org/Dotsies.ttf";
```

Comment:
Unfortunately, the author does'nt answer on [twitter](https://twitter.com/erictapen/status/885264205112377344). There seems to be no activity since 2013.

As I can't ask him wether redistribution is allowed and nixpkgs needs a stable URL, I'll close that issue. If the author by chance answers, I will reopen it.